magma-feedstock/recipe/bld.bat

```diff
@@ -1,12 +1,24 @@
+md build
+if errorlevel 1 exit /b 1
+
 cmake ^
-  -H. ^
-  -Bbuild ^
-  -G"Ninja" ^
+  -S . ^
+  -B build
```

Comment:
I guess it doesn't matter if we're skipping windows, but should there be a caret here?